docker/env/Dockerfile

```diff
@@ -1,15 +1,25 @@
-FROM centos:7
+FROM python:3.8
 
-ADD install-prereqs.sh install-prereqs.sh
-RUN ./install-prereqs.sh docker
-RUN yum update -y \
-    && yum install -y https://repo.ius.io/ius-release-el7.rpm \
-    && yum install -y python36u python36u-libs python36u-devel python36u-pip
+RUN DEBIAN_FRONTEND=noninteractive apt update && \
```

Comment:
It's better to change `apt` to `apt-get`. Debian shows warning if you try to run `apt` in scripts.